### ISSUE TITLE
[FC] Battle Kid 2 - Mountain of Torment: Black Screen

### ISSUE THREAD

**Issue by Shideravan** [higan-emu/ares#1309]
**Describe the issue**
Black screen. Probably missing Mapper 71 (Camerica) support

**Rom Information**
```
  sha256: 9fc3c7d8a348308baec47aec403d4f9f529058c153d68e7fc7e92be9d8454257
  name:   [071] Battle Kid 2 - Mountain of Torment
```

**Comment by invertego** [comment]
This actually appears to be an UNROM 512 game (mapper 30) which should be working as of #1225.

#1554 implements mapper 71, adding support for the following games: https://nescartdb.com/search/advanced?ines_op=equal&ines=71

**Comment by LukeUsher** [comment]
Fixed in #1225
